tests/tck/features/lookup/LookUpTopN.feature

```diff
@@ -0,0 +1,292 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Push topN down IndexScan Rule
+
+  Background:
+    Given a graph with space named "nba"
+
+  Scenario: push topN down to IndexScan with order by
+    When profiling query:
+      """
+      LOOKUP ON player | ORDER BY $-.VertexID | Limit 5
+      """
+    Then the result should be, in any order:
+      | VertexID            |
+      | "Amar'e Stoudemire" |
+      | "Aron Baynes"       |
+      | "Ben Simmons"       |
+      | "Blake Griffin"     |
+      | "Boris Diaw"        |
+    And the execution plan should be:
+      | id | name             | dependencies | operator info             |
+      | 5  | DataCollect      | 9            |                           |
+      | 9  | TopN             | 10           |                           |
+      | 10 | Project          | 11           |                           |
+      | 11 | TagIndexFullScan | 0            | {"orderBy": {"pos": "0"}} |
+      | 0  | Start            |              |                           |
+    When profiling query:
+      """
+      LOOKUP ON player | ORDER BY $-.VertexID | Limit 5
```

Comment:
As mentioned in the comments above, there is no need to test `limit and orderBy check` separately